docs/user_guides/fs/transformation_functions.md

```diff
@@ -30,8 +36,11 @@ The `@udf` decorator in Hopsworks creates a metadata class called [`HopsworksUdf
 | datetime.date            |
 | datetime.time            |
 
+The `drop` argument is optional and specifies the input arguments to remove from the final output after all transformation functions are applied. By default, all input arguments are retained in the final transformed output. 
```

Comment:
to remove from the final output after all transformation functions are applied. By default, all input arguments are retained in the final transformed output. 

I think you need an example to explain this.

For example, for an on-demand transformation function attached a feature group:
....

Reply:
I see the example below. That's probably enough, but refer to it in the text here.

Reply:
Provided link to example in the text.